modules/tensor_mechanics/tests/combined_creep_plasticity/combined_stress_prescribed.i

```diff
@@ -76,19 +77,19 @@
     function = pressure
   [../]
   [./u_bottom_fix]
-    type = DirichletBC
+    type = PresetBC
     variable = disp_y
     boundary = bottom
     value = 0.0
   [../]
   [./u_yz_fix]
-    type = DirichletBC
+    type = PresetBC
```

Comment:
These are really just fine as DirichletBC.  Preset only matters when you set them to nonzero values.